documentation/DotNetCli.md

```diff
@@ -34,3 +51,11 @@ All analyzers regarding xml documentation can only run if the xml processing is
 
 to the `buildOptions` node of the project.json file. Note that this might cause additional CS1591 diagnostics to appear by the compiler.
 They can be suppressed in the ruleset file if necessary.
+
+For VS VS2017 / dotnet core 1.1.1 SDK 1.0.1 projects update `.csproj`
+```xml
+    <PropertyGroup>
+        ...
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
I will check that later - but if people are using vs-code then they don't have the luxury of a checkbox :)